server/athenian/api/align/queries/metrics.py

```diff
@@ -521,15 +523,16 @@ def collect(
         for request, goal_ids, calc_result in zip(
             self._requests, self._goal_ids, batch_calc_results,
         ):
-            for interval_i, interval in enumerate(request.time_intervals):
-                int_ = cast(Interval, interval)
-                team_metrics_res.setdefault(int_, {})
+            for intervals_i, intervals in enumerate(request.time_intervals):
+                team_metrics_res.setdefault(intervals, {})
                 for metric_i, metric in enumerate(request.metrics):
-                    team_metrics_res[int_].setdefault(metric, {})
+                    team_metrics_res[intervals].setdefault(metric, {})
                     for team_i, (team_filters, goal_id) in enumerate(zip(request.teams, goal_ids)):
-                        team_metrics_res[int_][metric][
+                        # collect all values for the intervals
```

Comment:
We have a terminology problem here. Intervals are everywhere and we need to fix that. Let's call them how they are called in Analytics: intervals and granularities/granularity values.

```suggestion
                        # collect all values for the granularities
```